scripts/pkg/build_templates/opensearch/deb/debian/postinst

```diff
@@ -37,11 +37,17 @@ if ! grep -q '## OpenSearch Performance Analyzer' ${config_dir}/jvm.options; the
    echo "--add-opens=jdk.attach/sun.tools.attach=ALL-UNNAMED" >> ${config_dir}/jvm.options
 fi
 
-# Set owner
-chown -R opensearch.opensearch ${product_dir}
+# Set ownership and permissions
+# FIXME: the opensearch service should not have w permission in the config directory
```

Comment:
Write permission has to be there for the security demo script to run and make changes.

Reply:
I see you keep the write for others now.

Reply:
I recursively remove the r and x bit for others, so the onwer has rwx permission and the group has rx permission.  Ideally, the files should be owner by root so that the service can not rewrite them, but on first start the service want to create `opensearch.keystore` so for now I left the owner to `opensearch:opensearch`.